alta-back-end/src/main/java/com/alta/web/entity/ExamCreationRequest.java

```diff
@@ -0,0 +1,6 @@
+package com.alta.web.entity;
+
+import java.util.List;
+
+public record ExamCreationRequest(String title, List<Integer> studentsIds, List<Integer> tasksIds) {
+}
```

Comment:
I suppose it would be better to call it more simple, for instance just: "ExamRequest", cause we may accept not only creation requests in future.

Reply:
I dont mind